src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -215,6 +216,29 @@ public static Tag parseTag(String tag) throws ParseException {
         return new Tag(trimmedTag);
     }
 
+    /**
+     * Parses a {@code String medicalTag} into a {@code MedicalQualificationTag}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code medicalTag} is invalid.
+     */
+    public static MedicalQualificationTag parseMedicalTag(String medicalTag) throws ParseException {
+        requireNonNull(medicalTag);
+        String skillset;
+        String level;
+        try {
+            String[] trimmedList = medicalTag.trim().split(" ");
+            skillset = trimmedList[0];
+            level = trimmedList[1];
+        } catch (ArrayIndexOutOfBoundsException ex) {
+            throw new ParseException(MedicalQualificationTag.MESSAGE_CONSTRAINTS);
+        }
+        if (!MedicalQualificationTag.isValidQualification(level)) {
+            throw new ParseException(MedicalQualificationTag.MESSAGE_CONSTRAINTS_SKILL);
+        }
+        return new MedicalQualificationTag(skillset, level);
```

Comment:
Might be good to check for null values inside `MedicalQualificationTag` just in case `skillSet` is not specified. You can potentially add an assertion here as well.

```suggestion
        return new MedicalQualificationTag(skillSet, level);
```